osu.Game.Rulesets.Catch/CatchRuleset.cs

```diff
@@ -51,7 +51,7 @@ public class CatchRuleset : Ruleset, ILegacyRuleset
             new KeyBinding(InputKey.X, CatchAction.MoveRight),
             new KeyBinding(InputKey.Right, CatchAction.MoveRight),
             new KeyBinding(InputKey.Shift, CatchAction.Dash),
-            new KeyBinding(InputKey.Shift, CatchAction.Dash),
+            new KeyBinding(InputKey.Space, CatchAction.Dash),
```

Comment:
I'm not really sure if "Space" is good here

Reply:
It's probably fine.